src/test/java/com/microsoft/graph/functional/OneNoteTests.java

```diff
@@ -49,6 +50,10 @@
     private OnenotePage testPage;
     private OnenoteSection testSection;
     private SectionGroup testSectionGroup2;
+    /**
+     * The encoding type for getBytes
+     */
+    static final String JSON_ENCODING = "UTF-8";
```

Comment:
Again, investigate what encoding is appropriate. The field name JSON_ENCODING suggests you haven't though about it.

Reply:
Changed it to HTML_ENCODING. The W3C recommends UTF-8 as the default encoding in XML and HTML (reference : https://en.wikipedia.org/wiki/UTF-8).